semantic-python/semantic-python.cabal

```diff
@@ -58,59 +58,60 @@ library
   exposed-modules:
     Language.Python
     Language.Python.AST
-    Language.Python.Core
+    -- Language.Python.Core
```

Comment:
Can we file a bug to reenable the Core stuff (or elide it entirely for the time being)?

Reply:
https://github.com/github/semantic/issues/569